netengine/backends/snmp/airos.py

```diff
@@ -400,23 +400,74 @@ def RAM_free(self):
         free = self.get_value('1.3.6.1.4.1.10002.1.1.1.1.2.0')
         return int(free)
 
+    @property
+    def RAM_buffered(self):
+        """
+        Returns the buffered RAM of the device
+        """
+        buffered = self.get_value('1.3.6.1.4.1.10002.1.1.1.1.3.0')
+        return int(buffered)
+
+    @property
+    def RAM_cached(self):
+        """
+        Returns the cached RAM of the device
+        """
+        cached = self.get_value('1.3.6.1.4.1.10002.1.1.1.1.4.0')
+        return int(cached)
+
+    @property
+    def load(self):
+        """
+        Returns an array with load average values respectively in the last
+        minute, in the last 5 minutes and in the last 15 minutes
+        """
+        one = int(self.get_value('1.3.6.1.4.1.10002.1.1.1.4.2.1.3.1'))
+        five = int(self.get_value('1.3.6.1.4.1.10002.1.1.1.4.2.1.3.2'))
+        fifteen = int(self.get_value('1.3.6.1.4.1.10002.1.1.1.4.2.1.3.3'))
```

Comment:
It will be more efficient to get all three values in one request. Can you please do that?

Reply:
Done :+1: